### PR TITLE
Rename to rqt_gauges

### PR DESCRIPTION
**Pull request by ahcorde**
Rename package from `rqt_gauges_2` to `rqt_gauges`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # gauges2
```
Comment (caguero) [inline README.md]: Should we rename the header as well?

**Review by caguero — COMMENTED** [review]
I removed all the `build`, `install` and `log` directories and recompiled following the instructions but I got the following error:

```
caguero@cold:~/gauges_ws$ ros2 run rqt_gauges rqt_gauges
RosPluginProvider._parse_plugin_xml() plugin file "/home/caguero/gauges_ws/install/rqt_gauges_2/share/rqt_gauges_2/plugin.xml" in package "rqt_gauges_2" not found
RosPluginProvider._parse_plugin_xml() plugin file "/home/caguero/gauges_ws/install/rqt_gauges_2/share/rqt_gauges_2/plugin.xml" in package "rqt_gauges_2" not found
qt_gui_main() found no plugin matching "rqt_gauges.speedometer.Speedometer"
try passing the option "--force-discover"
[ros2run]: Process exited with failure 1
```

File: README.md (hunk 1 above)
Comment (ahcorde) [reply README.md]: https://github.com/ToyotaResearchInstitute/gauges2/pull/23/commits/e22589c36778147fc4e9b1c676ab864a65b58de2

**Review by caguero — APPROVED** [review]
All working passing `--force-discover`.

**Pull request merged**